nyt.py

```diff
@@ -272,6 +272,15 @@ def data_to_puz(puzzle):
             else:
                 rebus.add_rebus(None)
 
+    # See if any grid squares are marked up with circles
+    if any(x['type'] == 2 for x in data['cells'] if 'type' in x):
```

Comment:
Use of magic constant `2`. Would you consider refactoring this to a named constant such as `NYT_TYPE_CIRCLED`? That would make it easier to search for other uses (such as line 281), and to change/edit/extend as necessary in the future (e.g. if the type turns out to be a bitmask).

Reply:
Agreed, I'll make this change after the PR, including all of the possibilities for values